packages/form-core/tests/FieldApi.test-d.ts

```diff
@@ -32,7 +32,7 @@ it('should type value when nothing is passed into constructor', () => {
   })
 
   assertType<string | undefined>(field.state.value)
-  assertType<'name'>(field.options.name)
```

Comment:
I think `field.option.name` should be `'name'` only and that's also true on the code, maybe this change was lost there during a refactor? (same for the two instances below on this file).

I see the test is still passing with `assertType<'name'>`